### DISCUSSION TITLE
Reporting RCE Vulnerability

### DISCUSSION THREAD

**Discussion by rotil** [fief-dev/fief#221] [Ideas]
Hello,

I hope this email finds you well. I recently discovered a Remote Code
Execution (RCE) vulnerability on the https://fief.fief.dev/ server.
I believe it is crucial to bring this to your attention as it poses a
significant security risk.

Unfortunately, I couldn't find a dedicated security email address to
report this vulnerability. Therefore, I kindly request guidance on how I
can securely share the detailed information with your team.

I want to ensure that this vulnerability is addressed promptly and
effectively, and I am committed to assisting in any way possible to
support your efforts in securing the server.

Thank you for your attention to this matter, and I look forward to your
guidance on how to proceed.

Best regards,

Behnam Yazdanpanah

**Comment by fief-bailiff** [comment]
Hail, @rotil 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Hi @rotil,

Thank you for your message and your willing to share your findings with us.

You can share the details securely from the Security tool of GitHub: https://github.com/fief-dev/fief/security/advisories/new
